tasks/configureApache.yml

```diff
@@ -4,37 +4,83 @@
 - name: prepare all required variables
   set_fact:
     httpd_conf_file: "{{ apache_dir }}/{{ ( ansible_os_family == 'Debian' ) | ternary( 'apache2','conf/httpd' ) }}.conf"
-    ssl_conf_file: '{{ apache_dir }}/conf.d/ssl.conf'
+    ssl_conf_file: "{{ apache_dir }}/{{ ( ansible_os_family == 'Debian' ) | ternary( 'conf-available','conf.d' ) }}/ssl.conf"
 
 - name: ensure that httpd.conf is correct
   become: True
   template:
     src: "{{ ( ansible_os_family == 'Debian' ) | ternary( 'apache2','httpd' ) }}.conf.j2"
```

Comment:
Why set in play? This should be done by included vars.

Reply:
Done